docker/ssh_data/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && \
 # add a user 'geoserver' with default password : 'geoserver'
 # that can be overriden by environement variable $USER_PASS
 RUN groupadd --gid 999 geoserver
-RUN useradd -ms /bin/bash --home /home/geoserver -p wtw.aC5paV.qU --uid 999 --gid 999 geoserver
+RUN useradd -ms /bin/bash --home /home/geoserver -p $(perl -e'print crypt("${USER_PASS:-geoserver}", "salt")') --uid 999 --gid 999 geoserver
```

Comment:
Perl command is not working. Single quotes disable variable substitution so `${USER_PASS:-geoserver}` is not replaced. Instead, you can use echo to generate perl code and pipe result into perl :
```bash
RUN useradd -ms /bin/bash --home /home/geoserver -p $(echo "print crypt("${USER_PASS:-geoserver}", "salt")" | perl) --uid 999 --gid 999 geoserver
```